GNUmakefile

```
@@ -41,7 +42,7 @@ lint-examples-sh: tools ## Run shell linter on examples (same as fmt but without
 	$(GOBIN)/shfmt -l -s -d ./examples
 
 SERVICE ?= gitlab-ce
-GITLAB_TOKEN ?= ACCTEST1234567890123
+GITLAB_TOKEN = ACCTEST1234567890123
```

This would prevent running tests against an external instance.
```suggestion
GITLAB_TOKEN ?= ACCTEST1234567890123
```

---

Was more a convenience change for myself, because I sometimes have that variable set for another purpose 🙈 . 
I'll revert this, if it's a use case :)

---

Oh, gotcha. Maybe we can rename it in this or a future PR then. (It's referenced in the testing section of CONTRIBUTING.md fyi).

---

Hey!

---

oups 😖  that wasn't on purpose. Changed it back ;)